src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/actions/ModifyQueueCapacityAction.java

```diff
@@ -119,6 +121,17 @@ public ResourceEnum getThreadPool() {
     return threadPool;
   }
 
+  //persist values into SQL tuple
+  private void persistValues() {
+    setRowValue(ActionField.RESOURCE_TYPE_FIELD.getField(), threadPool.getNumber());
+    setRowValue(ActionField.NODE_ID_FIELD.getField(), esNode.getNodeId().toString());
+    setRowValue(ActionField.NODE_IP_FIELD.getField(), esNode.getHostAddress().toString());
```

Comment:
A single action can impact multiple nodes in future - like `moveShard` actions. These fields should allow for multiple node values